Backend/src/main/java/com/abhinternship/auctionapp/controller/BidController.java

```diff
@@ -23,4 +24,9 @@ public BidController(BidService bidService) {
     public List<Bid> getBidsByProductId(@PathVariable Long productId) throws RepositoryException {
         return bidService.getAllByProductId(productId);
     }
-}
+
+    @GetMapping("/user")
+    public List<Bid> getByUser(@RequestParam(value = "user") User user, @RequestParam(value = "pageSize") Long pageSize) throws RepositoryException {
```

Comment:
*getBidsByUser. Also, why do you need the whole user object, just pass the userId?